resources/qml/pages/UserSettingsPage.qml

```diff
@@ -25,7 +25,9 @@ Rectangle {
         palette: Nheko.colors
         ScrollBar.horizontal.visible: false
         anchors.fill: parent
-        anchors.margins: Nheko.paddingLarge
+        //anchors.margins: Nheko.paddingLarge // padding right of the scrollbar is ugly anyway
+        anchors.topMargin: (collapsed? backButton.height : 0) + Nheko.paddingMedium
+        anchors.leftMargin: collapsed? Nheko.paddingMedium : Nheko.paddingLarge
```

Comment:
You are missing the bottom padding. That makes scrolling down weird.

Reply:
![image](https://user-images.githubusercontent.com/16312324/154825355-448aa5b8-0997-4cae-a16d-f9506b25c3d5.png)

Reply:
I didn't want the borders, but some distance of the last item from the bottom makes sense.